examples/triton/docker-compose-with-web.yml

```diff
@@ -0,0 +1,37 @@
+version: '2.1'
+
+services:
+  redis:
+    image: autopilotpattern/redis:3.2.8-r1.0.0
+    mem_limit: 4g
+    restart: always
+    labels:
+      - triton.cns.services=redis
+    environment:
+      - CONTAINERPILOT=file:///etc/containerpilot.json
+      - affinity:com.docker.compose.service!=~redis
+    env_file: _env
+    network_mode: bridge
+
+  consul:
+    image: autopilotpattern/consul:0.7.2-r0.8
+    command: >
+      /usr/local/bin/containerpilot
+      /bin/consul agent -server
+        -bootstrap-expect 1
+        -config-dir=/etc/consul
+        -ui-dir /ui
+    restart: always
+    mem_limit: 128m
+    labels:
+      - triton.cns.services=redis-consul
+    network_mode: bridge
+
+  webserver:
+    image: jasonpincin/ap-redis-example-webserver:1.0.0
+    restart: always
+    mem_limit: 128m
+    ports:
+      - 8000
```

Comment:
The example app was hard-coded to run on `8000`, so that it worked locally. I could change the app to accept a PORT env var and listen on that, then pass that in... it complicates the example a bit, so I didn't bother. You have another preference?